datasets/us/wy/med_exclusions/us_wy_med_exclusions.yaml

```diff
@@ -0,0 +1,40 @@
+title: US Wyoming Medicaid Provider Exclusion List
+entry_point: crawler.py
+prefix: us-medwy
+coverage:
+  frequency: daily
+  start: "2024-09-24"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Medical providers deemed ineligible to participate in Wyoming State's Medicaid program.
+description: |
+  The Wyoming Department of Health maintains a list of providers that have been excluded from
+  the state Medicaid program. Providers listed are not allowed to receive reimbursement for
+  Medicaid services in any capacity.
+publisher:
+  name: Wyoming Department of Health
+  description: >
+    > Wyoming Medicaid is a joint federal and state government program that pays for medical
+    > care for some low-income and medically needy individuals and families.
```

Comment:
let's try and make the description more about Wyoming Department of Health https://health.wyo.gov/admin/about/